.github/workflows/linux-build-and-test.yaml

```diff
@@ -221,18 +221,24 @@ jobs:
     - name: Stack trace
       if: always() && steps.collectlogs.outputs.coredumps == 'true'
       run: |
-        sudo coredumpctl gdb <<<"
+        sudo coredumpctl debug --debugger=gdb --debugger-arguments='' <<<"
```

Comment:
Could you also adjust https://github.com/timescale/timescaledb/blob/main/.github/workflows/linux-32bit-build-and-test.yaml ? Having the same output for stack traces in all actions would be great.

Reply:
Done.